### PR TITLE
Cleanup trade wallet code

### PR DESCRIPTION
**Pull request by chimp1984**

### CODE REVIEW

File: core/src/main/java/bisq/core/btc/model/RawTransactionInput.java [hunk 1]
```diff
@@ -27,12 +27,16 @@
 
 import javax.annotation.concurrent.Immutable;
 
+/**
+ * Holds the relevant data for the connected output for a tx input.
+ * ParentTransaction means here the spending tx, the tx of the output not the parentTransaction of the input itself.
+ * Also index refers to the index of the spending output
+ */
```
Comment (ghubstan) [inline core/src/main/java/bisq/core/btc/model/RawTransactionInput.java]: I would reformat the method javadoc with some minor grammar changes:
```
    /**
     * Holds the relevant data for the connected output for a tx input.
     * @param index  the index of the parentTransaction
     * @param parentTransaction  the spending output tx, not the parent tx of the input
     * @param value  the number of satoshis being spent
     */

```

Comment (ghubstan) [reply core/src/main/java/bisq/core/btc/model/RawTransactionInput.java:34]: But `ACK`, it's only a comment.

**Review by ghubstan — DISMISSED** [review]

File: core/src/main/java/bisq/core/btc/model/RawTransactionInput.java (hunk 1 above)
Comment (chimp1984) [reply core/src/main/java/bisq/core/btc/model/RawTransactionInput.java]: Thanks, will do a commit a bit later (in the middle of some other works atm...)

File: core/src/main/java/bisq/core/btc/wallet/TradeWalletService.java [hunk 2]
```diff
@@ -1111,15 +1111,7 @@ public Transaction traderSignAndFinalizeDisputedPayoutTx(byte[] depositTxSeriali
...
         Script redeemScript = get2of2MultiSigRedeemScript(buyerPubKey, sellerPubKey);
-
-        Script hashedMultiSigOutputScript = get2of2MultiSigOutputScript(buyerPubKey, sellerPubKey,
-                hashedMultiSigOutputIsLegacy);
-
         Coin msOutputValue = buyerPayoutAmount.add(sellerPayoutAmount).add(txFee);
-        TransactionOutput hashedMultiSigOutput = new TransactionOutput(params, null, msOutputValue, hashedMultiSigOutputScript.getProgram());
-        Transaction depositTx = new Transaction(params);
-        depositTx.addOutput(hashedMultiSigOutput);
-
```
Comment (ghost) [inline core/src/main/java/bisq/core/btc/wallet/TradeWalletService.java:1122]: :+1: @chimp1984 thanks for finding this!  :sweat_smile:

**Review by ghost — DISMISSED** [review]

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**